en/airframes/airframe_reference.md

```diff
@@ -401,6 +406,25 @@ This page lists all supported airframes and types including
 </tr>
 </tbody></table>
 
+### Quadrotor h
+
+<div>
+<img src="../../assets/airframes/types/QuadRotorH.svg" width="29%" style="max-height: 180px;"/>
+</div>
+
+<table style="width: 100%; table-layout:fixed; font-size:1.5rem;">
+ <colgroup><col style="width: 30%"><col style="width: 70%"></colgroup>
+ <thead>
+   <tr><th>Name</th><th></th></tr>
+ </thead>
+<tbody>
+<tr id="copter_quadrotor_h_reaper_500_quad">
```

Comment:
There are no "common" outputs here. That is correct based on the [config file](https://github.com/PX4/Firmware/blob/master/ROMFS/px4fmu_common/init.d/4040_reaper), but given this is the only case, should we perhaps update that init file and regenerate?

Reply:
Yep, I added the outputs